### DISCUSSION TITLE
How to use SQL query vendor trace information ?

### DISCUSSION THREAD

**Discussion by lbknxy** [google/perfetto#948] [Q&A] [answered]
We usually capture systrace file with soc vendor trace , such as kgsl_xxx:

```
...

threadName-22210   (  19171) [006] ..... 71152.506330: kgsl_mem_alloc: gpuaddr=0x401785a000 size=4096 tgid=19171 usage=cl_buffer_map id=797 flags=0x8c0c0b00
threadName-22210   (  19171) [006] ..... 71152.506332: kgsl_mem_mmap: useraddr=0x7687472000 gpuaddr=0x401785a000 size=4096 usage=cl_buffer_map id=797 flags=0x8c0c0b00
threadName-22210   (  19171) [006] ..... 71152.506350: kgsl_mem_alloc: gpuaddr=0x401785b000 size=4096 tgid=19171 usage=cl_buffer_map id=798 flags=0x8c0c0b00
threadName-22210   (  19171) [006] ..... 71152.506358: kgsl_mem_mmap: useraddr=0x7687400000 gpuaddr=0x401785b000 size=4096 usage=cl_buffer_map id=798 flags=0x8c0c0b00

....
```

I want to know which SQL table these trace information will resident ? 
I have tried ftrace_event and args table, But got nothing.

**Comment by LalitMaganti** [comment, marked as answer]
If you are parsing systrace files directly into Perfetto, only events explicitly supported in https://cs.android.com/android/platform/superproject/main/+/main:external/perfetto/src/trace_processor/importers/systrace/systrace_line_parser.cc;l=90 are parsed. Any other events are not parsed and are simply dropped.

Please switch to collecting Perfetto traces if you want this functionality.
